packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -64,10 +64,11 @@ type UUIColorPickerSize = 'small' | 'medium' | 'large';
  */
 @defineElement('uui-color-picker')
 export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
-  @query('[part="input"]') _input!: UUIInputElement;
-  @query('.color-picker__preview') _previewButton!: HTMLButtonElement;
-  @query('#swatches') _swatches!: UUIColorSwatchesElement;
+  @query('[part="input"]') private _input!: UUIInputElement;
+  @query('.color-picker__preview') private _previewButton!: HTMLButtonElement;
+  @query('#swatches') private _swatches!: UUIColorSwatchesElement;
 
+  @state() private _value: string = '';
   @state() private inputValue = '';
```

Comment:
I know this is not a new property, but we use it in some of the new code. What is the purpose of storing the intermediary value here rather than setting it directly to `this.value` throughout the code?

Reply:
I wanted to add some get and set logic for `this.value` with the purpose of firing the `setColor()` method, in case of the color is being changed from the outside.

Reply:
Yes, but this is `inputValue` that I was asking about. It seems to just hold an intermediary value coming from `syncValues`.

Reply:
I recall the `inputValue` value is the typed value in input field, which may not be the hex(a) code, it can be rgba(a) or color name as well, while the synced values is the converted color IIRC.

E.g. `inputValue` can be `black`, while the synced value it `#000000`.

Reply:
Makes sense, @bjarnef, thanks